core/modules/catalog-next/helpers/cacheProductsHelper.ts

```diff
@@ -0,0 +1,24 @@
+import { products } from 'config'
+
+export const prefetchStockItems = (cachedProductsResponse, cache = {}) => {
+  const skus = []
+  let prefetchIndex = 0
+  cachedProductsResponse.items.map(i => {
+    if (products.configurableChildrenStockPrefetchStatic &&
```

Comment:
does this actually belong here? because this just supposed to limit the prefetching of children. but configurableChildrenStockPrefetchStatic = false would prevent prefetching as a whole.